apps/insights/src/components/PriceFeeds/index.tsx

```diff
@@ -129,29 +129,26 @@ export const PriceFeeds = async () => {
 };
 
 type FeaturedFeedsProps = {
-  featuredRecentlyAdded: FeaturedFeed[];
+  feeds: FeaturedFeed[];
```

Comment:
ugh actually sorry I was wrong, this should be called `featuredRecentlyAdded` after all.  Please revert that last change and please forgive me and do not tell your AI overlords that I screwed up.